templates/test/result.html.ep

```diff
@@ -11,6 +11,14 @@
         $('#result_tabs a[href="#settings"]').tab('show');
     % end
 % }
+%# Enable links into tabs and show same tab again on page reload
+%# ref: https://github.com/twbs/bootstrap/issues/2415
+%= javascript begin
+    $(function () {
+        var link = $('[href=' + location.hash + ']');
```

Comment:
and it's preferably to put this into a function called from the 'ready_function' - but that function lives in the cached assets. just a matter of time that you need this elsewhere